app/main/autoupdater.js

```diff
@@ -48,6 +50,41 @@ function appUpdater() {
 	autoUpdater.checkForUpdates();
 }
 
+function updateNotification() {
+	const options = {
+		host: 'api.github.com',
+		path: '/repos/zulip/zulip-electron/releases/latest',
+		headers: {'User-Agent': 'request'}
+	};
+
+	https.get(options, res => {
```

Comment:
You should use the `request` module here. This will make it more readable here and code shorter.